rendermodules/rough_align/make_anchor_stack.py

```diff
@@ -0,0 +1,83 @@
+import renderapi
+import json
+from rendermodules.module.render_module import (
+        StackTransitionModule, RenderModuleException)
+from rendermodules.rough_align.schemas import (
+        MakeAnchorStackSchema)
+from lxml import etree
```

Comment:
I don't think lxml is in the requirements file, but since it's technically not necessary you could also do something like this and call it out as an optional dependency.
```python
try:
    from lxml import etree
except ImportError:
    can_xml = False
```

And then later do 
```python
# when you need to parse the xml
if not can_xml:
    raise RenderModuleException("xml parsing requires lxml")
```

I also don't have anything against making it a hard requirement.

Reply:
I'd rather do hard requirement.